docs/meta/supported-os.md

```diff
@@ -6,31 +6,30 @@ nav_order: 2
 has_children: false
 parent: Information
 search_exclude: false
-last_modified_date: 2023-02-27
+last_modified_date: 2023-12-25
 ---
 
 # Supported Operating Systems
 
-## Full Support:
-* Non-EOL, consumer versions of Windows 10/11 only:
+## Fully Supported
+* All actively maintained versions of Windows 10 and Windows 11, provided they are not end-of-life (EOL) and are consumer editions:
     * Home
     * Pro
+    * Pro Education
```

Comment:
Pro Education is separate from Education. See https://en.wikipedia.org/wiki/Windows_10_editions#Organizational_editions